src/components/learner-credit-management/AssignmentDetailsTableCell.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Hyperlink } from '@edx/paragon';
+
+import { configuration } from '../../config';
+
+const AssignmentDetailsTableCell = ({ row, enterpriseSlug }) => {
+  const { ENTERPRISE_LEARNER_PORTAL_URL } = configuration;
+  return (
+    <>
+      <div className="font-weight-bold">{row.original.learnerEmail}</div>
```

Comment:
[context] We are assuming that `learnerEmail` always exists here for now, though this may change in the future once product requirements around PII scrubbing (i.e., retirement pipeline) is better understood. In the future, if we need to account for missing learner email here, we can rely on the `EmailAddressTableCell` component which has messaging already in place for this case.